site/src/not-found.md

```diff
@@ -0,0 +1,15 @@
+---
+title: '404: Page not found'
+description: '404: Page not found'
+template: standard
+---
+
+<div class="padding-lg border border-radius-lg">
+	<h1 id="four-zero-four" style="font-size: 12rem" class="font-family-monospace" aria-describedby="four-zero-four-message">404</h1>
```

Comment:
Can you help me understand the reasoning for this `aria-describedby` here?

Reply:
Just thinking that 404 as a standalone h1 was a bit non-descript

Reply:
I tried to keyboard navigate to it with NVDA and Narrator. It just reads "heading level 1 404".

Reply:
I'm going vote that it's not necessary here - since like Ming showed, it's not guaranteed, and even if folks did get the description, they'd get that same content on the next element.